cli/test/schema.spec.js

```diff
@@ -31,90 +33,83 @@ indexes = ["datetime"]
 template = "any()a;lkdfjlakjf;ladkfjal;kfj"
 `
 
-parse_schema(testSchema)
-const currentOpts = {};
+const valid_file_system = {
+  '.hz': {
+    'schema.toml': testSchema,
+  },
+};
+
+const currentOpts = { };
+
 describe("hz schema", () => {
-  before(() => {
-    // Start RethinkDB server
-    return new Promise((resolve, reject) => {
-      start_rethinkdb_server({
-        driverPort: 38015,
-        bind: ["127.0.0.1"],
-        dataDir: tmpdir() + "/horizon-test",
-        db: "horizon_schema_test"
-      }).then((server) => {
-        currentOpts.dbPort = server.driverPort;
-        resolve();
-      }).catch((err) => {
-        console.log(err)
-      });
-    });
-  });
+  let rdb_server;
+  let rdb_conn;
+
+  before('start rethinkdb', () =>
+    start_rdb_server({
+      bind: ["127.0.0.1"],
+      dataDir: tmpdir() + "/horizon-test",
```

Comment:
One thing I forgot to add here was we should save this generated path and then remove it after the tests are done or have exited.